client-js/queryEditor/QueryEditor.js

```diff
@@ -118,6 +119,16 @@ class QueryEditor extends React.Component {
     })
   }
 
+  formatQuery = () => {
+    // select * from table
+    const { query } = this.state
+    var result = sqlFormatter.format(query.queryText)
+    this.state.query.queryText = result
```

Comment:
This can be simplified to `query.queryText = result`. Even though it is declared with `const` above properties of an object can be changed.